src/disco/components/InfoDialog.js

```diff
@@ -0,0 +1,37 @@
+import React, { PropTypes } from 'react';
+import translate from 'core/i18n/translate';
+
+import 'disco/css/InfoDialog.scss';
+
+export class InfoDialog extends React.Component {
+  static propTypes = {
+    addonName: PropTypes.string.isRequired,
+    closeAction: PropTypes.func.isRequired,
+    imageURL: PropTypes.string.isRequired,
+    i18n: PropTypes.object.isRequired,
+  }
+
+  render() {
+    const { addonName, closeAction, i18n, imageURL } = this.props;
+    return (
+      <div className="show-info" role="dialog"
+           aria-labelledby="show-info-title" aria-describedby="show-info-description">
+        <div className="info">
+          <div className="logo">
+            <img src={imageURL} alt="" width="48" height="48" />
```

Comment:
Good question. They provide a layout hint [1] but I think moving this to the CSS is just as 
good [2] so I'll do that (plus it's more consistent with what we've already done).

[1] http://jsbin.com/basixugebe/edit?html,css,js,output 
[2] http://jsbin.com/qofife/edit?html,css,js,output